pilot/pkg/bootstrap/server.go

```diff
@@ -1183,7 +1185,56 @@ func (s *Server) initMeshHandlers() {
 	})
 }
 
-func (s *Server) initWorkloadTrustBundle() {
+func (s *Server) addIstioCAToTrustBundle(args *PilotArgs) {
+	var err error
+
+	if s.CA != nil {
+		// When CA is enabled, fetch TrustBundle from CA
+		rootCerts := []string{string(s.CA.GetCAKeyCertBundle().GetRootCertPem())}
+		err = s.workloadTrustBundle.UpdateTrustAnchor(&tb.TrustAnchorUpdate{
+			TrustAnchorConfig: tb.TrustAnchorConfig{Certs: rootCerts},
+			Source:            tb.SourceIstioCA,
+		})
+		if err != nil {
+			log.Errorf("fatal: unable to add CA root as trustAnchor")
```

Comment:
nit: if its truly fatal, do log.Fatal, which will exit. If its not, don't log "fatal"